test/clt-tests/installation/rhel-release-update.rec

```diff
@@ -0,0 +1,41 @@
+––– input –––
+yum install -y --skip-broken mariadb mariadb105 which > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+yum install -y -q https://repo.manticoresearch.com/manticore-repo.noarch.rpm > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+yum install -y -q manticore manticore-extra > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+searchd --version
+––– output –––
+Manticore %{SEMVER} %{COMMITDATE} (columnar %{SEMVER} %{COMMITDATE}) (secondary %{SEMVER} %{COMMITDATE})
+Copyright (c) 2001-2016, Andrew Aksyonoff
+Copyright (c) 2008-2016, Sphinx Technologies Inc (http://sphinxsearch.com)
+Copyright (c) 2017-%{YEAR}, Manticore Software LTD (https://manticoresearch.com)
+––– input –––
+manticore-executor -v
+––– output –––
+PHP %{SEMVER} (cli) (built: #!/[a-zA-Z]{3}/!# #!/[0-9]+/!# %{YEAR} %{TIME}) (ZTS)
+Copyright (c) The PHP Group
+Zend Engine #!/v[0-9]+\.[0-9]+\.[0-9]+/!#, Copyright (c) Zend Technologies
+––– block: start-searchd-release –––
+––– block: check-buddy –––
+––– block: check-backup –––
+––– input –––
+yum install -y -q --enablerepo manticore-dev manticore manticore-extra > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+searchd --version
+––– output –––
+Manticore %{SEMVER} #!/[a-z0-9]{7,9}@[0-9]{6}/!# dev
+Copyright (c) 2001-2016, Andrew Aksyonoff
+Copyright (c) 2008-2016, Sphinx Technologies Inc (http://sphinxsearch.com)
+Copyright (c) 2017-%{YEAR}, Manticore Software LTD (https://manticoresearch.com)
+WARNING: Error initializing columnar storage: daemon requires columnar library v%{NUMBER} (trying to load v%{NUMBER})
+WARNING: Error initializing secondary index: daemon requires secondary library v%{NUMBER} (trying to load v%{NUMBER})
```

Comment:
This is not expected.

Reply:
@PavelShilin89 Please don't mark a comment as resolved without:
* any change in the code
* comment

as it's confusing.

Reply:
Done